src/datumaro/components/annotation.py

```diff
@@ -1372,19 +1373,19 @@ class Cuboid2D(Annotation):
     [(x1, y1), (x2, y2), (x3, y3), (x4, y4), (x5, y5), (x6, y6), (x7, y7), (x8, y8)].
 
 
-      6---7
+      2---3
      /|  /|
-    5-+-8 |
-    | 2 + 3
+    1-+-4 |
+    | 5 + 6
     |/  |/
-    1---4
+    8---7
```

Comment:
I understood the overall structure, is there any reason to change the bottom and top face?

Reply:
I aligned the order of points with Kitti format which describes top face first.